doofinder-for-woocommerce/includes/api/endpoints/class-endpoint-product.php

```diff
@@ -257,6 +258,41 @@ private static function get_categories( $data ) {
 		return $data;
 	}
 
+	/**
+	 * Processes product categories and adds a new `category_merchandising` field
+	 * containing relative URLs for each product category.
+	 *
+	 * @param array $data The product data to process.
+	 * @return array Product data including the new `category_merchandising` field.
+	 * @throws WP_Error If there is an error retrieving term links.
+	 * @since 2.7.6
+	 */
+	private static function get_category_merchandising( $data ) {
+		if ( isset( $data['categories'] ) ) {
+			$data['category_merchandising'] = array();
+			foreach ( $data['categories'] as $category ) {
+				if ( empty( $category['id'] ) || ! is_numeric( $category['id'] ) ) {
+					continue;
+				}
+
+				$term_link = get_term_link( (int) $category['id'], 'product_cat' );
+				if ( is_wp_error( $term_link ) ) {
+					continue;
+				}
+
+				$components    = wp_parse_url( $term_link );
+				$relative_link = isset( $components['path'] ) ? $components['path'] : '';
+
+				if ( isset( $components['query'] ) && ! empty( $components['query'] ) ) {
```

Comment:
```suggestion
				if ( ! empty( $components['query'] ) ) {
```

I think the `isset()` can be omitted here since empty prevents any failure, even if it is not set. Internally, it hides the errors with an `@`